test/integration/default/controls/app-engine.rb

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id = attribute('project_id', required: true, type: :string)
+domain = attribute('domain', required: true, type: :string)
+region = attribute('region', type: :string, default: ENV['TF_VAR_region'])
```

Comment:
Never depend on environment variables for InSpec attributes - it makes the input sources weird.